src/content/decentralized-social-networks.md

```diff
@@ -0,0 +1,92 @@
+
+Decentralized social media website exist on a peer-to-peer network comprising thousands of nodes around the globe. Even if some nodes fail, the network will run uninterrupted, making applications resistant to failures and outages. 
+
+Using decentralized storage systems like the [InterPlanetary File System (IPFS)](https://ipfs.io/), Ethereum-based social networks protect user information from exploitation and malicious use. No one will sell your personal information to advertisers, neither will hackers be able to steal your confidential details. 
+
+Many blockchain-based social platforms have native tokens that power monetization in absence of advertising revenue. Users can buy these tokens to access certain features, complete in-app purchases, or tip their favorite content creators. 
+
+## Decentralized social networks on Ethereum {#Ethereum-social-media}
+
+The Ethereum network has become the preferred tool for developers creating decentralized social media networks owing to the popularity of its tokens (ERC-20/ERC-721) and its massive user base. Here are some examples of Ethereum-based social networks:
+
+- Peepeth
+
+[Peepeth](https://peepeth.com/) is a blockchain-based microblogging site similar to Twitter. The dApp runs on the Ethereum blockchain and uses the InterPlanetary File System (IPFS) to store user data.
+
+Users can send short messages called "Peeps", which cannot be deleted or modified. You can collect tips or tip anyone on the platform in ether (ETH) without leaving the app.
+
+- Mirror
+
+[Mirror](https://mirror.xyz/) is a decentralized and user-owned content distribution platform similar to Medium. Users can sign up by connecting their wallets or providing an Ethereum Name Service (ENS) username.
+
+Posts published on Mirror are permanently stored on Arweave, a decentralized storage platform. Users can earn money by selling their articles as [non-fungible tokens (NFTs)](https://ethereum.org/en/nft/).
+
+- MINDS
+
+[MINDS](https://www.minds.com/) is considered one of the more popular blockchain social media platforms out there. It works like Facebook, and has racked up millions of users already.
+
+Users use the platform's native ERC-20 token $MIND to pay for items. You can also earn $MIND tokens by publishing popular content, contributing to the ecosystem, and accepting advertising offers.
+
+## Ethereum and legacy social media platforms {#Ethereum-and-legacy-social-media-networks}
+
+Decentralized social media platforms aren't the only ones trying to incorporate blockchain technology. Traditional sites are warming up to integrating Ethereum into their infrastructure.
+
+For instance, Reddit has [touted Community Points](https://cointelegraph.com/news/reddit-to-reportedly-tokenize-karma-points-and-onboard-500m-new-users), which are ERC-20 token that users can earn by posting quality content and contributing to the online community. You can redeem these tokens to gain exclusive privileges and perks.
```

Comment:
Should we mention they are working directly with Arbitrum here?

Reply:
Yeah, we could do that, if it's necessary (something like "Reddit is working with Arbitrum, a Layer-2 scaling platform for Ethereum"). I felt mentioning the relationship with Ethereum (i.e., using ERC-20 tokens) was enough.